src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
```

Comment:
Also possible to transfer the expression about the definition of the partner to a separate constant.
Example:

```suggestion
export const Person = ({ person }) => (
  const partnerRole = person.sex === 'm' ? 'wife' : 'husband';
  
  <section className="Person">
```